.github/workflows/interface-unit-tests.yml

```diff
@@ -93,7 +98,8 @@ jobs:
             "core-tests": 5,
             "gradients-tests": 2,
             "jax-tests": 5,
-            "tf-tests": 3
+            "tf-tests": 3,
+            "device-tests": 2
```

Comment:
This makes it so that up to 2 parallel runners can be used for device tests rather than just 1. cc @rashidnhm let me know if this is okay. This was easier to do than the other strategy i had in mind, which was to update the device tests job to run all tests inside a single job rather than splitting it into multiple jobs.

Reply:
No issues from my end on this